### ISSUE TITLE
How to install? 

### ISSUE THREAD

**Issue by bryanmccall** [meatpaste/ee-divert-docs#1]
I assume this installs in the standard way? But when copied to the addons folder, it does not show in the add-on screen to install. I'm using EE6.2.7.

**Comment by meatpaste** [comment]
extract the contents of the zip into your addon folder eg. `/system/user/addon/hinder`

**Comment by bryanmccall** [comment]
Yes. I've done that. But it is not listed on the add-on page in the control panel to click install. I assume it should be listed there?

**Comment by meatpaste** [comment]
Hi, the folder structure should look like this, let me know if anything is missing: https://pasteboard.co/GK6GhQnOv03G.png
